### PR TITLE
Components

### PR DESCRIPTION
**Pull request by IshmaelKargbo**
- Delete all the boilerplate from CRA (text, images, styles).
- Create a directory called components.
- Inside components, create a new Calculator.js file.
- Import it and use it in your main component (App.js).
- Test it in the browser; it should render without problems.

### CODE REVIEW

File: src/components/caculator.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  '1',
+  '2',
+  '3',
+  '+',
+  '0',
+  '.',
+  '=',
+];
+
+const Calculator = () => (
```
Comment (FrederickMih) [inline src/components/caculator.js]: - [x] Kindly make this component class-based. This is what the requirements demand.

File: src/App.js [hunk 2]
```diff
@@ -1,10 +1,13 @@
 import React from 'react';
 
 import './App.css';
+import Calculator from './components/caculator';
 
 const App = () => (
```
Comment (FrederickMih) [inline src/App.js]: - [x] Equally, you will need to make this component class-based to fulfill the requirements of this project and move a step toward approval.

**Review by FrederickMih — CHANGES_REQUESTED** [review]
Hi @IshmaelKargbo , 

You have done a fantastic job so far in this project!
There are some issues that you still need to work on to go to the next project but you are almost there!



# To Highlights :star: 

- Your code is well organized :+1: 
- PR has a title and summary :heavy_check_mark: 
- Linters are well set up :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 2 above)
Comment (IshmaelKargbo) [reply src/App.js]: I tried but I'm getting an error from react saying Component should be written as a pure function  react/prefer-stateless-function

File: src/components/caculator.js (hunk 1 above)
Comment (IshmaelKargbo) [reply src/components/caculator.js]: I tried but I'm getting an error from react saying Component should be written as a pure function react/prefer-stateless-function

**Comment by IshmaelKargbo** [comment]
I tried but I'm getting an error from react saying Component should be written as a pure function react/prefer-stateless-function

File: src/App.js (hunk 2 above)
Comment (FrederickMih) [reply src/App.js]: @IshmaelKargbo . Read about pure component and make use of it.

**Comment by FrederickMih** [comment]
> I tried but I'm getting an error from react saying Component should be written as a pure function react/prefer-stateless-function

@IshmaelKargbo, check out [this link](https://www.geeksforgeeks.org/reactjs-pure-components/) on how to use React pure component

**Pull request merged**